pipeline_dp/private_spark.py

```diff
@@ -0,0 +1,52 @@
+from pyspark import RDD
+from typing import Callable
+
+import pipeline_dp
+from pipeline_dp import aggregate_params, budget_accounting
+
+
+class PrivateRDD:
+    """A Spark RDD counterpart in PipelineDP guaranteeing that only anonymized data
+    within the specified privacy budget can be extracted from it through its API.
+    """
+
+    def __init__(self, rdd, budget_accountant, privacy_id_extractor):
+        self._rdd = rdd
+        self._budget_accountant = budget_accountant
+        self._privacy_id_extractor = privacy_id_extractor
+
+    def sum(self, sum_params: aggregate_params.SumParams) -> RDD:
+        """Computes DP sum.
+
+        Args:
+            sum_params: parameters for calculation
+        """
+
+        ops = pipeline_dp.SparkRDDOperations()
+        dp_engine = pipeline_dp.DPEngine(self._budget_accountant, ops)
+
+        params = pipeline_dp.AggregateParams(
+            metrics=[pipeline_dp.Metrics.SUM],
+            max_partitions_contributed=sum_params.max_partitions_contributed,
+            max_contributions_per_partition=
+                sum_params.max_contributions_per_partition,
+            low=sum_params.low,
+            high=sum_params.high,
+            public_partitions=sum_params.public_partitions)
+
+        data_extractors = pipeline_dp.DataExtractors(
+            partition_extractor=sum_params.partition_extractor,
+            privacy_id_extractor=self._privacy_id_extractor,
+            value_extractor=sum_params.value_extractor)
+
+        aggr_result = dp_engine.aggregate(self._rdd, params, data_extractors)
+        dp_result = ops.map_values(aggr_result, lambda acc: acc.compute_metrics())
```

Comment:
Please remove this line, just return the result of the `aggregate`(). 

Explanation: It will be in `aggregate()` soon (aggregate is not fully implemented yet :) ).

Reply:
Removed :)